examples/tsp/tsp.cc

```diff
@@ -89,7 +89,7 @@ auto main(int argc, char * argv[]) -> int
     auto succ = p.create_integer_variable_vector(n, 0_i, Integer(n - 1));
     auto dist = p.create_integer_variable_vector(n, 0_i, 745_i);
 
-    p.post(Circuit{succ, false, false});
+    p.post(Circuit{succ, false});
```

Comment:
I had a brief "uh oh does this mean I've been running with GAC turned on?" moment here. Remind me at some point to go and change all these bool parameters in function arguments into enums.